src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -64,6 +71,14 @@ public class TextureManager : MonoBehaviour
         /// </summary>
         private RenderTexture displayOutputTexture = null;
 
+        /// <summary>
+        /// Gets whether or not holograms should be rendered on a black background (which allows for post-processing in quadrant mode)
+        /// or against the video background (which allows for correct alpha blending for partially-transparent holograms).
+        /// </summary>
+        private bool IsVideoOutputQuadrantMode => Compositor != null && Compositor.VideoRecordingLayout == VideoRecordingFrameLayout.Quad;

```

Comment:
>IsVideoOutputQuadrantMode [](start = 21, length = 25)

This flag name is extremely confusing. From what I can tell, this flag is an indication of whether video recording is in quadrant mode compared to whether or not the capture card will output quadrant mode. Can we change this to IsVideoRecordingQuadrantMode?